aws-integration/README.md

```diff
@@ -0,0 +1,138 @@
+# Connect AWS X-Ray and CloudWatch to kyma
+
+## Overview 
+
+Sometimes you may want to transfer your logs, metrics and traces to other systems as well. Kyma provides a possibility to integrate it with some of the AWS components by deploying a custom OTEL Collector which would collect traces, logs, and metrics from kyma system and forward them to the AWS. In order to do this, we recommend you to use [AWS distribution](https://aws-otel.github.io) for OTEL Collector. 
+In this tutorial we will show you how to deploy OTEL Collector in order to connect Kyma observability 
+
+## Prerequisistes 
+
+- Kyma as the target deployment environment
+- AWS account with permissions to create new users and security policies
+- Helm 3.x if you want to deploy open-telemetry sample application
+
+## Installation
+
+### Preparation
+
+1. Export your Namespace as a variable. Replace `{NAMESPACE}` placeholder in the following command and run it:
+
+    ```bash
+    export KYMA_NS="{NAMESPACE}"
+    ```
+1. If you don't have created a Namespace yet, do it now:
+    ```bash
+    kubectl create namespace $KYMA_NS
+    ```
+1. Export the AWS region you want to use as variable. Replace `{NAMESPACE}` placeholder in the following command and run it:
```

Comment:
This is never used. Some intermediate stop to copy this into the secret is missing. Maybe just remove this step here and add it to your secret.